src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const spiderHTML = document.querySelector('.spider');
```

Comment:
It's better to use more descriptive variable names. Consider renaming 'spiderHTML' to 'spiderElement'